common/src/main/java/com/viaversion/viaversion/rewriter/EntityRewriter.java

```diff
@@ -510,6 +510,9 @@ public EntityType trackAndRewrite(final PacketWrapper wrapper, final int typeId,
         }
 
         final EntityType entityType = typeFromId(trackMappedType ? mappedTypeId : typeId);
+        if (entityType == null) {
+            return null;
+        }
         tracker(wrapper.user()).addEntity(entityId, entityType);
```

Comment:
Doesn't matter too much, but me want empty lines after braces if a "normal" statement of a "next" logical step follows